pymatgen/entries/__init__.py

```diff
@@ -72,18 +73,27 @@ def energy_per_atom(self) -> float:
     def __str__(self):
         return self.__repr__()
 
-    def normalize(self, mode: str = "formula_unit") -> None:
+    def normalize(self, mode: str = "formula_unit", inplace=True) -> None:
```

Comment:
Return signature here should be edited to be `-> Optional[Entry]` I believe (this is causing the mypy linter failure)